Xabe.FFmpeg/FFmpeg.cs

```diff
@@ -185,12 +185,18 @@ private void FindProgramsFromPath(string path)
             }
             FileInfo[] files = new DirectoryInfo(path).GetFiles();
 
-            FFprobePath = files.FirstOrDefault(x => x.Name.StartsWith(FFprobeExecutableName, true, CultureInfo.InvariantCulture))
+            FFprobePath = files.FirstOrDefault(x => x.Name.StartsWith(FFprobeExecutableName, true, CultureInfo.InvariantCulture) && !IsLibraryFile(x.Extension))
```

Comment:
Check if the file has no extension or .exe
We cannot exclude all other extensions